packages/admin/cms-admin/src/dam/DamTable.tsx

```diff
@@ -80,6 +82,7 @@ const Folder = ({ id, filterApi, ...props }: FolderProps) => {
                         </ToolbarItem>
                         <ToolbarFillSpace />
                         <ToolbarActions>
+                            <CurrentDamFolderProvider folderId={id}>{props.additionalToolbarItems}</CurrentDamFolderProvider>
```

Comment:
why is this provider around toolbar items only?

Reply:
I can't think of a reason. Fixed: d72ab27aad7f080473f694288c5ebc25524d3728